py3status/modules/diskdata.py

```diff
@@ -90,8 +90,11 @@ def post_config_hook(self):
                 self.init[name] = {"placeholders": placeholders, "keys": match}
 
         if self.init["diskstats"]:
-            self.last_diskstats = self._get_diskstats(self.disk)
-            self.last_time = time.monotonic()
+            try:
+                self.last_diskstats = self._get_diskstats(self.disk)
+                self.last_time = time.monotonic()
+            except Exception:
+                self.init["diskstats"] = []
```

Comment:
My bad. It should be an empty dict... It's definitely an empty list in any other modules.